doc/database.rst

```diff
@@ -12,7 +12,7 @@ Database API
 .. autoclass:: tango.DbDevExportInfo
     :members:
 
-.. autoclass:: tango.DbDevExportInfo
+.. autoclass:: tango.DbDevExportInfos
     :members:
 
 .. autoclass:: tango.DbDevImportInfo
```

Comment:
I see the `tango/__init__.py` also mentions the plural case of this one, so can we add a section for `tango.DbDevImportInfos` too, please?